app/src/layouts/adminLayout/adminHeader/adminHeader.scss

```diff
@@ -74,25 +28,25 @@
   font-size: 20px;
   text-transform: uppercase;
   display: inline-block;
-  @media (max-width: $SCREEN_XS_MAX){
+  @media (max-width: $SCREEN_XS_MAX) {
     font-size: 14px;
   }
 }
-.header-crumb{
+.header-crumb {
   display: none;
-  @media (max-width: $SCREEN_XS_MAX){
+  @media (max-width: $SCREEN_XS_MAX) {
     display: inline;
   }
 }
-.admin-header-controls{
+.admin-header-controls {
   float: right;
```

Comment:
Is float really necessary here? Is it possible to make the same with flex for example?

Reply:
Done